src/components/views/messages/TextualBody.tsx

```diff
@@ -418,6 +418,23 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
         });
     };
 
+    /**
+     * This acts as a fallback in-app navigation handler for any body links that
+     * were ignored as part of linkification because they were already links
+     * to start with (e.g. pills, links in the content).
+     */
+    private onBodyLinkClick = (e: MouseEvent): void => {
+        const target = e.target as Element;
+        if (target.nodeName !== "A" || target.classList.contains("linkified")) return;
```

Comment:
could the `linkified` className be imported from the linkify options? Or maybe a note added in the options file where the className is set that this depends on it.

Reply:
Ah yes, seems like a good idea, added this.